iocore/eventsystem/IOBuffer.cc

```diff
@@ -80,7 +80,7 @@ auto
 make_buffer_size_parser()
 {
   return [l = swoc::Lexicon<int>{
-            {{0, {"128"}},
+            {swoc::Lexicon<int>::Definition{0, {"128"}},
```

Comment:
I think this can be handled by `swoc::Lexicon` and not by us here. @SolidWallOfCode ?

Reply:
Talk to @cmcfarlen - it's a bit of an odd use.